torch/nn/modules/padding.py

```diff
@@ -28,7 +60,7 @@ class ConstantPad1d(Module):
     """
 
     def __init__(self, padding, value):
-        super(ConstantPad1d, self).__init__()
+        super(ConstantPad1d, self).__init__(padding, value)
```

Comment:
What's the value of subclassing these classes on `(_ConstantPadNd)` if you are not actually doing any code de-duplication.

Reply:
Also, we kept around these nn classes only for legacy purposes, after the F.pad refactor. Going forward, `F.pad` is the blessed and nice way to go forward.

Reply:
I originally used the torch/nn/modules/conv.py as a template, but you are right, I could just make each of these a module and get rid of _ConstantPadNd.

Reply:
Ok, so if nn.ConstantPad1d and nn.ConstantPad2d did not exist, should I remove those?  Also, should there be a message in the docs that the functional should be used instead of these modules so people know where to look if they want to use "nn.ConstantPad1/3d"?

Reply:
yes, remove them if they didn't exist, and for the other padding layers, add a not in the docs saying F.pad is preferred.